homeassistant/helpers/entity_registry.py

```diff
@@ -83,6 +83,15 @@ def async_is_registered(self, entity_id):
         """Check if an entity_id is currently registered."""
         return entity_id in self.entities
 
+    @callback
+    def async_get_entity_id(self, domain: str, platform: str, unique_id: str):
```

Comment:
`async_get_or_create` should use this function

Reply:
Done